all-collections/_our-team/meg-gutshall.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+
+first_name: Meg
+last_name: Gutshall
+list_order: H
+position: Open Source Advocate and Welcome Wagoner
```

Comment:
How about either `community relations` or `developer advocate`? The audience for the website is companies, foundations we're looking for grants from, and potential sponsors.

Reply:
How about site manager then? I don't know if I encompass either of those roles.

Reply:
Or maybe Open-Source Community Advocate?

Reply:
That works!